ml_metadata/tools/mlmd_bench/util.h

```diff
@@ -65,6 +76,17 @@ tensorflow::Status InsertNodesInDb(int64 num_artifact_nodes,
                                    int64 num_context_nodes,
                                    MetadataStore& store);
 
+// Generates random integer within the range of the specified `dist`.
+int64 GenerateRandomNumberFromUD(const UniformDistribution& dist,
+                                 std::minstd_rand0& gen);
+
+// Generates a categorical distribution with a Dirichlet prior with
+// `concentration_param`, the sample size of the returned discrete distribution
+// is specified by `sample_size`.
+std::discrete_distribution<int64>
+GenerateCategoricalDistributionWithDirichletPrior(int64 sample_size,
+                                                  int64 concentration_param);
```

Comment:
move to workload .cc, or add test coverage

Reply:
Moved to workload.cc. Plan to add test coverage in the next PR. Thanks!